src/components/integrated/OakLessonNavItem/OakLessonNavItem.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+import { OakLessonNavItem } from "./OakLessonNavItem";
+
+import { OakFlex } from "@/components/base";
+
+const meta: Meta<typeof OakLessonNavItem> = {
+  component: OakLessonNavItem,
+  tags: ["autodocs"],
+  title: "components/integrated/OakLessonNavItem",
+  argTypes: {
+    summary: {
+      control: {
+        type: "text",
+      },
+    },
+    videoLength: {
+      if: { arg: "lessonSectionName", eq: "video" },
```

Comment:
I wanted to conditionally show the `grade` and `numQuestion` controls only for `exit-quiz` and `starter-quiz` but this condition can only contain one value 🤬